crates/red_knot_test/Cargo.toml

```diff
@@ -13,7 +13,7 @@ license.workspace = true
 [dependencies]
 red_knot_python_semantic = { workspace = true, features = ["serde"] }
 red_knot_vendored = { workspace = true }
-ruff_db = { workspace = true, features = ["testing"] }
+ruff_db = { workspace = true, features = ["os", "testing"] }
```

Comment:
It's not clear to me what changed that the `os` feature is now required or is this a feature we already missed declaring as dependency before?